Ghana/GHA_transform_load_dlt.ipynb

```diff
@@ -0,0 +1,160 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 0,
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {
+      "byteLimit": 2048000,
+      "rowLimit": 10000
+     },
+     "inputWidgets": {},
+     "nuid": "8fd4f596-b73b-4a74-a3d7-eebb43599352",
+     "showTitle": false,
+     "tableResultSettingsMap": {},
+     "title": ""
+    }
+   },
+   "outputs": [],
+   "source": [
+    "import dlt\n",
+    "from pyspark.sql.functions import substring, col, lit, when, initcap, element_at, split, upper, trim, lower, regexp_replace, regexp_extract, substring, expr, concat, coalesce\n",
+    "\n",
+    "TOP_DIR = \"/Volumes/prd_mega/sboost4/vboost4\"\n",
+    "INPUT_DIR = f\"{TOP_DIR}/Documents/input/Countries\"\n",
+    "WORKSPACE_DIR = f\"{TOP_DIR}/Workspace\"\n",
+    "COUNTRY_MICRODATA_DIR = f'{WORKSPACE_DIR}/microdata_csv/Ghana'\n",
+    "\n",
+    "CSV_READ_OPTIONS = {\n",
+    "    \"header\": \"true\",\n",
+    "    \"multiline\": \"true\",\n",
+    "    \"quote\": '\"',\n",
+    "    \"escape\": '\"',\n",
+    "}\n",
+    "\n",
+    "@dlt.expect_or_drop(\"year_not_null\", \"YEAR IS NOT NULL\")\n",
+    "@dlt.table(name=f'gha_boost_bronze')\n",
+    "def boost_bronze():\n",
+    "    bronze_df = (spark.read\n",
+    "      .format(\"csv\")\n",
+    "      .options(**CSV_READ_OPTIONS)\n",
+    "      .option(\"inferSchema\", \"true\")\n",
+    "      .load(COUNTRY_MICRODATA_DIR)\n",
+    "    )\n",
+    "    for old_col_name in bronze_df.columns:\n",
+    "        new_col_name = old_col_name.replace(\" \", \"_\")\n",
+    "        bronze_df = bronze_df.withColumnRenamed(old_col_name, new_col_name)\n",
```

Comment:
Is this rename really needed? I don't see any referenced column from bronze that has _ in it. Bronze really should have the data in its rawest form so I'd suggest moving this rename to silver, if this rename is indeed necessary.